presidio-anonymizer/README.MD

```diff
@@ -1 +1,126 @@
-# TODO add the document Omri wrote.
\ No newline at end of file
+# Presidio anonymizer
+
+## Description
+
+The Presidio anonymizer is a Python based module for anonymizing detected PII text
+entities with desired values.
+
+Presidio anonymizer comes by default with the following anonymizers:
```

Comment:
replace anonimizers with "transformations" @omri374 WDYT?

Reply:
I don't have a strong preference here :) Maybe just for the sake of having it similar to the analyzer where we have the analyzer and different recognizers.

Reply:
ok, lets close for now